_component_design/assisted-search.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Assisted Search
+parent: Search and Filters
+layout: component
+category: Components
+usage: >
+  Use assisted search when dealing with a complex data set.  Assisted search provides users the ability to see available facets or filters within the search field, allowing for custom construction of search criteria before execution. 
+preview-image: preview-images/assisted-search.svg
+resource: true
+status: stable
+last-modified: 2017-11-07
+---
+
+{% include toc.html %}
+
+## Assisted search
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Default state
+
+Upon clicking the search box the user should be presented with a list of keyword that they can use to filter their search. The user can click or type out a keyword to use it as a filter to narrow down their search.
```

Comment:
When the user clicks the search box, show them a list of keywords that they can use to filter their search.

Reply:
updated.